Modules/Microsoft365DSC/DSCResources/MSFT_EXOMailboxStatistics/MSFT_ExoMailboxStatistics.schema.mof

```diff
@@ -0,0 +1,74 @@
+[ClassVersion("1.0.0.0"), FriendlyName("EXOMailboxStatistics")]
+class MSFT_EXOMailboxStatistics : OMI_BaseResource
+{
```

Comment:
The resource should have at least one Key parameter. This is currently causing failing QA tests.